apis/networking/v1beta1/apiproduct_types.go

```diff
@@ -89,11 +89,26 @@ func (a *APISelector) APINamespacedName() types.NamespacedName {
 	return types.NamespacedName{Namespace: a.Namespace, Name: name}
 }
 
-type InfraRateLimitSpec struct {
+type AuthRateLimitSpec struct {
 	MaxValue int32 `json:"max_value"`
 	Period   int32 `json:"period_in_seconds"`
 }
 
+type PreAuthRateLimitSpec struct {
+	MaxValue int32 `json:"max_value"`
+	Period   int32 `json:"period_in_seconds"`
```

Comment:
I've left a couple comments about these in #46.

Reply:
Can we reuse the struct already defined?

Reply:
Sure, I will propagate the changes

Reply:
> Can we reuse the struct already defined?

we can. I deliberately duplicated the struct to make explicit they are different things and can have different structure and fields. If the design gets stabilized, I will remove duplication (if it exists)